6.Oop/Lesson-4/Task_1/Building.cs

```diff
@@ -0,0 +1,82 @@
+// 1. Реализовать класс для описания здания (уникальный номер здания, высота, этажность, количество квартир, подъездов).
+//  Поля сделать закрытыми, предусмотреть методы для заполнения полей и получения значений полей для печати.
+//  Добавить методы вычисления высоты этажа, количества квартир в подъезде, количества квартир на этаже и т.д. 
+// Предусмотреть возможность, чтобы уникальный номер здания генерировался программно. 
+// Для этого в классе предусмотреть статическое поле, которое бы хранило последний использованный номер здания, 
+// и предусмотреть метод, который увеличивал бы значение этого поля.
+
+namespace Task_1
+{
+    public sealed class Building
+    {
+        private static int _lastNumber;
+
+        private int _number;
+
+        private int _floorCount;
+
+        private double _height;
+
+        private int _flatCount;
+
+        private int _entranceCount;
+
+        private static int GenerateNumber() => _lastNumber++;
```

Comment:
```suggestion
        private static int GenerateNumber() => ++_lastNumber;
```
Идентификаторы не должны начинаться с 0.